test/windows_test.go

```diff
@@ -0,0 +1,225 @@
+package test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/Azure/dalec"
+	gwclient "github.com/moby/buildkit/frontend/gateway/client"
+	moby_buildkit_v1_frontend "github.com/moby/buildkit/frontend/gateway/pb"
+)
+
+func TestWindows(t *testing.T) {
+	t.Parallel()
+
+	ctx := startTestSpan(baseCtx, t)
+	testWindows(ctx, t, "windowscross/container")
+}
+
+func testWindows(ctx context.Context, t *testing.T, buildTarget string) {
```

Comment:
What's the difference between the existing test function and this one?

Reply:
It removes the TestSpecs from the third test, and I think there was one other change. Since we can't run commands in the final container, there's no way to run the tests from the spec. Is there a way around that?

Reply:
Maybe we can re-use the other test function but have a spec generator with an option elide tests from the generated spec.

Reply:
Oh, there is also the issue that some of the test specfiles have runtime dependencies, which windows outputs can't have. I will look at re-using the other test function.